RobotArdu/src/main/java/de/fhg/iais/roberta/worker/compile/ArduinoCompilerWorker.java

```diff
@@ -92,9 +95,15 @@ public void execute(Project project) {
                 scriptName = compilerResourcesDir + "arduino-resources/build_project_rob3rta.sh";
                 break;
             case "festobionic":
+            case "festobionicflower":
                 boardVariant = "esp32";
                 arduinoVariant = "ARDUINO_ESP32_DEV";
-                scriptName = compilerResourcesDir + "arduino-resources/build_project_festobionic.sh";
+                mmcu = "NOTUSED";
```

Comment:
this feature (compiling on windows machines) is very welcome, but here only implemented for festobionic.
Please add it later in the code, that all other systems can benefit from it.